### ISSUE TITLE
Training doesn't seem to change the predictions.

### ISSUE THREAD

**Issue by KasparPeterson** [atselousov/transformer_chatbot#15]
Hello!
Thank you for your great work. I have been trying to understand and use this model for some experimentation. 

The model trained for over 17hours on V100, the loss has been decreasing and everything seems to be fine. I used a small dataset of 28k lines of conversation data.

![Screenshot 2019-05-08 at 10 52 53](https://user-images.githubusercontent.com/3907242/57359231-dc297300-717f-11e9-941e-9c146c15a5ec.png)

However, when using: `python interactive.py` the model predictions don't reflect my small dataset. It seems like everything is overridden by the previous checkpoint.

Is it possible to do transfer learning on top of your checkpoint? Should I train for longer?

**Comment by shachi04** [comment]
@KasparPeterson Did you figure out a way to solve this? I am also trying to finetune the model further on my dataset but I seem to be having the same problem. How many epochs did you train it for? 

@atselousov : Any thoughts ? My dataset has about 49k lines of dialog data. I am training it for 100 epochs currently on top of your checkpoint (last_checkpoint).

**Comment by KasparPeterson** [comment]
I am afraid not :/
